qiskit/tools/jupyter/version_table.py

```diff
@@ -14,6 +14,8 @@
 """A module for monitoring backends."""
 
 import time
+from sys import modules
+from collections import OrderedDict
```

Comment:
Do you need an ordered dict? For all our supported versions of Python all dicts are insertion ordered now.

Reply:
Fixed in d0d858872d82501187b71b2611e8d769cb40d5dc